ch01.asciidoc

```diff
@@ -43,11 +43,15 @@ We'll be seeing them throughout this book:
 consumer::
 A regular consumer on the Internet or in the offline world who wants to make purchases.
 
+Alice runs a small coffee shop in an upmarket street in Berlin.  She knows about bitcoin and wants to accept it in her shop, but has been reluctant to do so because she knows that bitcoin payments take approx. 10 minutes to be confirmed into her account, and that the fees that her customer needs to pay for a bitcoin payment are variable.  With the lightning network, she knows that her regular clients, such as Beppe can pay for their coffee at her shop, and she knows that the bitcoin is instantly in her wallet, and that the fees for her clients are small enough not to concern her clients. 
```

Comment:
This is much nicer formulated that the merchant that I put into it but is it any different than the merchant?
What I like about it is that she has the Bitcoin internal viewpoint of "I already want to use bitcoin, but it needs to get better"

Reply:
Agreed - should be under merchant - I'll move it.  My intention was to match the format of "Mastering Bitcoin", where user stories/use cases are introduced in the first chapter and then expanded upon in the later chapters (a format idea that I really liked) - Is that how you see it also?

Reply:
```suggestion
Alice runs a small coffee shop in an upmarket street in Berlin.  She knows about Bitcoin and wants to accept it in her shop, but has been reluctant to do so because she knows that Bitcoin payments take approx. 10 minutes to be confirmed into her account, and that the fees that her customer needs to pay for a Bitcoin payment are variable. This is something that most costumers would not be pleased with. However with the Lightning Network, she knows that her regular clients, such as Beppe can pay for their coffee at her shop, quickly and with negligible fees. Additionally, by using the Lightning Network, Alice has all funds deposited instantly to her wallet and with usually smaller fees on her side as well. Ultimately this allows Alice to provide a better service or to offer better pricing for her producst.
```

Reply:
Bigger changes on this paragraph. I like the idea but it doesn't really close. Is Bitcoin/LN just another payment method? The advantages need to be evident imo. As such I basically re-wrote the last 2/3 sentences so feel free to re-iterate.

Lastly is this a real story like Dennis below? I ask because if it isn't then Beppe should be changed to Bob (the usual exemple) or to Charlie? (to keep the order Alice, Berlin, C...).